packages/atlas/src/components/_inputs/Checkbox/Checkbox.styles.ts

```diff
@@ -1,82 +1,63 @@
 import { css } from '@emotion/react'
-import styled from '@emotion/styled/'
```

Comment:
**FYI** this caused me a lot of trouble. I had weird errors from babel in storybook and it turned out this was the reason...